documentation/content-standards.md

```diff
@@ -51,39 +51,39 @@ To give an example, an entry titled "What is Three.js?" should have the file nam
 - All text should be written in Markdown
   - For more details on Markdown, see Codecademy's [Curriculum Markdown Style Guide](http://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/).
 - Each subsection should begin with a heading of size h2 (i.e. preceded by `##`)
-- All in-line code should be delineated by single backticks (`)
+- All in-line code should be delineated by single backticks (\`)
 
 #### Code Blocks
 
 Code examples are an excellent teaching tool. Many term entries will contain a code example as a standard field. Encyclopedia entries are more flexible, but if your encyclopedia entry doesn't contain any code blocks, take a moment to ask yourself if one would help!
 
-All code blocks should be delineated by triple backticks (```)
-
-- C: ```c
-- C#: ```cs
-- C++: ```cpp
-- CSS: ```css
-- Error: ```error
-- Emojicode: ```emojic
-- HTML: ```html
-- Java: ```java
-- JavaScript: ```js
-- PHP: ```php
-- Pseudo: ```pseudo
-- Python: ```py
-- R: ```r
-- Ruby: ```rb
-- Sass: ```scss
-- Shell/Program Output: ```shell
-- SQL: ```sql
+All code blocks should be delineated by triple backticks (\`\`\`)
```

Comment:
I presume this in intentional?

Reply:
gonna be real I'm very surprised that this file [renders reasonably](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md) as currently written, but I didn't check before making these changes. I guess they're not necessary 🤷